largestPrimeFactor.js

```diff
@@ -1,6 +1,15 @@
 exports.largestPrimeFactor = function(n) {
   let primeNumber = 0;
-  // do your work here
+  let numArr = [];
+
+  for (let i = 2; i <= n; i++) {
+    while (n % i === 0) {
+      numArr.push(i);
+      n /= i;
+    }
+  }
+  let highNum = numArr[numArr.length - 1];
```

Comment:
You don't need the `highNum` variable since it's just being set to `primeNumber` right below this line.